lightning/webelement.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import annotations
+
+from base64 import b64decode
+from typing import Optional
+
+from lightning_adapter.api.elements_api import ElementsApi
+from lightning_adapter.api.screenshots_api import ScreenshotsApi
+from lightning_adapter.models import LocatorStrategy, EmptyRequest, FindElementRequest,\
+                                      FindElementResponseValue, Rect, ElementSendKeysRequest
+
+from .exceptions import WebDriverException
+from .window import Position, Size
+
+
+class Locator:
+
+    def __init__(self, expression: str, strategy: LocatorStrategy):
+        self._expression = expression
+        self._strategy = strategy
+
+    @property
+    def expression(self) -> str:
+        return self._expression
+
+    @property
+    def strategy(self) -> LocatorStrategy:
+        return self._strategy
+
+
+class By(Locator):
+
+    @staticmethod
+    def css_selector(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.CSS_SELECTOR)
+
+    @staticmethod
+    def xpath(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.XPATH)
+
+    @staticmethod
+    def tag_name(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.TAG_NAME)
+
+    @staticmethod
+    def link_text(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.LINK_TEXT)
+
+    @staticmethod
+    def partial_link_text(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.PARTIAL_LINK_TEXT)
+
+
+class Accessibility(dict):
+    def __init__(self, elements_instance: ElementsApi, session_id: str, element_id: str):
+        self._elements_instance = elements_instance
+        self._session_id = session_id
+        self._element_id = element_id
+
+    @property
+    def role(self) -> str:
+        _role = self._elements_instance\
+            .get_element_computed_role(session_id=self._session_id, element_id=self._element_id).value
+        return _role
+
+    @property
+    def label(self) -> str:
+        _label = self._elements_instance\
+            .get_element_computed_label(session_id=self._session_id, element_id=self._element_id).value
+        return _label
+
+    def __repr__(self):
+        _c = dict(role=self.role, label=self.label)
+        return repr(_c)
+
+
+class WebElement(dict):
+
+    def __init__(self, elements_instance: ElementsApi, element_id: str, session_id: str):
+        self._elements_instance = elements_instance
+        self._element_id = element_id
```

Comment:
Renamed `_element_id` -> `_id`.

Reply:
fixed in 990528b5d10e02f0199fddd9a570dc33e95fa248